phys2bids/tests/test_integration.py

```diff
@@ -0,0 +1,466 @@
+
+    # Check sampling frequency
+    started_idx = [log_idx for log_idx, log_str in enumerate(
+                      log_info) if 'Sampling started' in log_str]
+    started_found = log_info[started_idx[0]]
+    assert '10.425007798392297' in started_found
+
+    # Check start time
+    start_idx = [log_idx for log_idx, log_str in enumerate(
+                      log_info) if 'first trigger' in log_str]
+    start_found = log_info[start_idx[0]]
+    assert 'Time 0' in start_found
+
+    # Checks json file
+    with open(os.path.join(test_path, 'Test_belt_pulse_samefreq.json')) as json_file:
+        json_data = json.load(json_file)
+
+    # Compares values in json file with ground truth
+    assert json_data['SamplingFrequency'] == 10000.0
+    assert json_data['StartTime'] == 10.425007798392297
+    assert json_data['Columns'] == ['time', 'RESP - RSP100C', 'PULSE - Custom, DA100C',
+                                    'MR TRIGGER - Custom, HLT100C - A 5', 'PPG100C', 'CO2', 'O2']
+
+    # Remove generated files
+    for filename in glob.glob(os.path.join(test_path, 'phys2bids*')):
+        os.remove(filename)
+    for filename in glob.glob(os.path.join(test_path, 'Test_belt_pulse_samefreq*')):
+        os.remove(filename)
+
+
+def test_integration_multifreq():
+    """
+    Does the integration test for a multi-frequency file
+    """
+
+    url = 'https://osf.io/9a7yv/download'
+    test_path = resource_filename('phys2bids', 'tests/data')
+    test_filename = 'Test_belt_pulse_multifreq.acq'
+    test_full_path = os.path.join(test_path, test_filename)
+    wget.download(url, test_full_path)
+    test_chtrig = 3
+
+    phys2bids(filename=test_filename, indir=test_path, outdir=test_path,
+              chtrig=test_chtrig, num_timepoints_expected=1)
+
+    # Check that files are generated
+    assert os.path.isfile(os.path.join(test_path, 'Test_belt_pulse_multifreq_625.0.log'))
+    assert os.path.isfile(os.path.join(test_path, 'Test_belt_pulse_multifreq_625.0.json'))
+    assert os.path.isfile(os.path.join(test_path, 'Test_belt_pulse_multifreq_625.0.tsv.gz'))
+    assert os.path.isfile(os.path.join(test_path, 'Test_belt_pulse_multifreq_trigger_time.png'))
+    assert os.path.isfile(os.path.join(test_path, 'Test_belt_pulse_multifreq_10000.0.log'))
+    assert os.path.isfile(os.path.join(test_path, 'Test_belt_pulse_multifreq_10000.0.json'))
+    assert os.path.isfile(os.path.join(test_path, 'Test_belt_pulse_multifreq_10000.0.tsv.gz'))
+
+    """
+    Checks 625 Hz output
+    """
+    # Read log file of frequency 625 (note that this file is not the logger file)
+    with open(os.path.join(test_path, 'Test_belt_pulse_multifreq_625.0.log')) as log_info:
+        log_info = log_info.readlines()
+
+    # Check timepoints expected
+    expected_idx = [log_idx for log_idx, log_str in enumerate(
+                      log_info) if 'Timepoints expected' in log_str]
+    expected_found = log_info[expected_idx[0]]
+    assert '1' in expected_found
```

Comment:
Same comment as above tests: creating a helper function or making this a for-loop would be great!